sdk/instrumentation/opentelemetry-instrumentation-azure-sdk/src/transformations.ts

```diff
@@ -93,3 +93,16 @@ export function toSpanOptions(spanOptions?: InstrumenterSpanOptions): SpanOption
     links,
   };
 }
+
+/**
+ * A helper function that converts a raw string to its boolean representation.
+ *
+ * This function treats "false" and "0" as false, but otherwise uses truthy-ness of the value.
+ *
+ * @param rawValue - The raw string value to convert to boolean.
+ * @returns - true if the {@link rawValue} is truthy, false otherwise.
+ */
+export function toBoolean(rawValue?: string): boolean {
```

Comment:
This loosely matches what dotnet does https://cs.github.com/Azure/azure-sdk-for-net/blob/1645ea51d73e7b4ec240cbdb88f99ae2039a7ccb/sdk/core/Azure.Core/src/DefaultClientOptions.cs#L18 so we're consistent with other languages